packages/site/src/components/social-meta-tags.tsx

```diff
@@ -0,0 +1,33 @@
+import { formatFieldName } from "@framework/system/src/util/string-utils"
+
+type SocialMetaTagsProps = {
+	title: string
+	siteName: string
+}
+
+export function SocialMetaTags({ siteName, title }: SocialMetaTagsProps) {
+	const description = `Search, compare, and discover top ${formatFieldName(
```

Comment:
I thought this description is supposed to be in `packages/site/src/data/descriptions.ts` ?